packages/markdown/src/components/Code.tsx

```diff
@@ -0,0 +1,26 @@
+import { List } from "@alloy-js/core";
+import { Children } from "@alloy-js/core/jsx-runtime";
+
+export interface CodeProps {
+  /** Language of the code block */
+  lang?: string;
+
+  /** Content of the codeblock, use instead of children. */
```

Comment:
```suggestion
  /** Content of the code block */
```

Was a little confusing because it feels like its saying, don't use children...

That said, this does the same thing as setting children to the string you pass to content, so I'm not sure this prop is worth having? I could see the argument though. If you like it, fix up the docstring and merge!